src/containers/AudioUploader/components/AudioContent.jsx

```diff
@@ -7,59 +7,50 @@
  */
 
 import React, { Fragment } from 'react';
-import PropTypes from 'prop-types';
 import { injectT } from '@ndla/i18n';
+import { useFormikContext } from 'formik';
 import AudioPlayer from './AudioPlayer';
 import FormikField from '../../../components/FormikField';
 
-const AudioContent = ({ t, values, setFieldValue }) => (
-  <Fragment>
-    <FormikField
-      label={t('form.title.label')}
-      name="title"
-      title
-      noBorder
-      placeholder={t('form.title.label')}
-    />
-    {!values.id && (
-      <FormikField noBorder name="audioFile" label={t('form.audio.file')}>
-        {() => (
-          <input
-            id="audioFile"
-            name="audioFile"
-            type="file"
-            onChange={evt => {
-              setFieldValue(
-                'filepath',
-                evt.target && evt.target.files[0]
-                  ? URL.createObjectURL(evt.target.files[0])
-                  : undefined,
-              );
-              setFieldValue('audioFile', evt.target.files[0]);
-            }}
-          />
-        )}
-      </FormikField>
-    )}
-    {(values.id || (values.audioFile && values.filepath)) && (
-      <AudioPlayer audio={values.audioFile} filepath={values.filepath} />
-    )}
-  </Fragment>
-);
+const AudioContent = ({ t }) => {
+  const { values, setFieldValue } = useFormikContext();
 
-AudioContent.propTypes = {
-  classes: PropTypes.func.isRequired,
-  values: PropTypes.shape({
-    id: PropTypes.number,
-    filepath: PropTypes.string,
-    audioFile: PropTypes.shape({
-      fileSize: PropTypes.number,
-      language: PropTypes.string,
-      mimeType: PropTypes.string,
-      url: PropTypes.string,
-    }),
-  }),
-  setFieldValue: PropTypes.func.isRequired,
+  return (
+    <Fragment>
+      <FormikField
+        label={t('form.title.label')}
+        name="title"
+        title
+        noBorder
+        placeholder={t('form.title.label')}
+      />
+      {!values.id && (
+        <FormikField noBorder name="audioFile" label={t('form.audio.file')}>
```

Comment:
Defaulter ikke (Formik)Field til input? 
altså kan du ikke bare ha en FormikField komponent også sende alle propsene til input til den?

Reply:
Har ikke sett nøye på koden i denne filen annet enn å prøve å bruke context for lettere å gjenbruke. 
Denne inputen tar inn en fil da, vet du om det da kan gjøres på samme måte i Field?

Reply:
nei. det vet jeg ikke. Bare vet at input er default :P